CMakeLists.txt

```diff
@@ -145,6 +145,10 @@ if(WIN32 AND ONEMKL_SYCL_IMPLEMENTATION STREQUAL "dpc++")
   set(CMAKE_CXX_COMPILE_OBJECT "<CMAKE_CXX_COMPILER> -fsycl /nologo <DEFINES> <INCLUDES> /EHsc <FLAGS> /Fo<OBJECT> -c <SOURCE>")
   set(CMAKE_CXX_CREATE_STATIC_LIBRARY "lib /nologo <OBJECTS> /out:<TARGET>")
   set(CMAKE_CXX_LINK_EXECUTABLE "<CMAKE_CXX_COMPILER> -fsycl /nologo <OBJECTS> -o <TARGET> <LINK_LIBRARIES>")
+  find_library(MKL_SYCL_LIB NAMES mkl_sycl
+    HINTS $ENV{MKLROOT} ${MKL_ROOT}
+    PATH_SUFFIXES lib/intel64)
+  set(CMAKE_CXX_CREATE_SHARED_LIBRARY "<CMAKE_CXX_COMPILER> -fsycl /nologo <OBJECTS> ${MKL_SYCL_LIB} /link /out:<TARGET> /implib:<TARGET_IMPLIB> /pdb:<TARGET_PDB> /dll /version:<TARGET_VERSION_MAJOR>.<TARGET_VERSION_MINOR> <LINK_FLAGS> <LINK_LIBRARIES>")
```

Comment:
can you fix the indentation?

Reply:
Curious, why do we need this special flag for lapack example and not for unit tests? (assuming unit tests were working fine?)

Reply:
Do you mean indentation in line 149/150? They have two more spaces because they are continuation of the `find_library` function on line 148
As for unit_tests, I'm seeing problem in lapack/getrf on windows (the test hangs for me, the cause of this problem is still in investigation). Therefore unit_test could have the same "no kernel.." problem as examples

Reply:
Oh, you are right about indentation. My bad. 

Regarding the need for flag in examples: @ericlars do you have any feedback?